File: binaryTreeHeight.py
class Node:
      def __init__(self,info): 
          self.info = info  
          self.left = None  
          self.right = None 
           

       // this is a node of the tree , which contains info as data, left , right
'''

def height(root, count = -1):
    if root is None: 
        return count
    count += 1
    leftHeight = height(root.left, count)
    rightHeight = height(root.right, count)
    return max(leftHeight, rightHeight)

File: binaryTreeHeight.js
// Definition for a binary tree node
function TreeNode(val) {
  this.val = val;
  this.left = this.right = null;
}

// Add any helper functions you may need here


function visibleNodes(root) {
  // Write your code here
  return rec(root, 0)
  
  function rec(root, level) {
    if (!root) {
      return level
    }
    

    let left = rec(root.left, level + 1)
    let right = rec(root.right, level + 1)
    
    return Math.max(left, right)
  }
}
